<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="output">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>761</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="vitsconfirm_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>合成</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="moegoepath_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_sovitsintro">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>使用so-vits模型进行音色转换</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_savepath_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>保存为</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="moegoepathbutton_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savepathbutton_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>250</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输入音频：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_savepath_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>inference_main.py 路径：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="savepathline">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputpath">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="inputpathbutton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_autopitch">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>自动音高</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pitch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>音高调整：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_pitch">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_slice_db">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice db:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_slice_db">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-40</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_noise_level">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>噪音级别：</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_noise">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
